### PR TITLE
#172990691 Landing Page

### PR DESCRIPTION
**Pull request by the22mastermind**
#### What does this PR do?

Add the landing page

#### Description of Task to be completed?
- Install required libraries
- Create NavBar component
- Update Home component
- Write tests

#### How should this be manually tested?
- In your terminal, run `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-frontend.git` to clone this repo

- Run `git fetch origin ft-landing-page-172990691:ft-landing-page-172990691` to pull this PR's branch

- Run `git checkout ft-landing-page-172990691` to switch to the fetched branch above

- Run `yarn install` to install all dependencies

- Start development server by running `yarn dev`.

- The server should launch the app in your default browser at `localhost:{{PORT}}`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
  [#172990691](https://www.pivotaltracker.com/story/show/172990691)

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export default function Card({ title, items }) {
+  return (
+    <Paper elevation={3} className="card">
+      <div className="card-title">{title}</div>
+      <div className="testimony-bar mb24">
+        <MdRemove color="#00A799" size={32} />
+      </div>
+      <div className="card-items">
+        {
+          items.map((item) => <a className="card-item" href="/" key={item}>{item}</a>)
```
Comment (kipropbrian) [inline src/components/Card.js]: Put a proper key here. Like the key in the array.

File: src/components/HelloComponent.js [hunk 2]
```diff
@@ -1,43 +1,66 @@
...
-    return (
-      <div>
-        <h1>
-          {customMessage.welcomeToBarefootNomad}
-        </h1>
-        <h2>{message}</h2>
-        <button type="button" onClick={displayMessage}>
-          try redux
-        </button>
+const HelloComponent = () => (
```
Comment (descholar-ceo) [inline src/components/HelloComponent.js]: Why did you call this a HelloComponent? In my opinion, you should have renamed HomeComponent or LandingPageComponent or something in that way

**Review by descholar-ceo — COMMENTED** [review]
@the22mastermind Congrat on a great job that you finally did! However, I have a small concern, this landing page seems to not fit well on devices with a small screen, could you work on that? Look at the screenshot below
Screenshot from 2020-05-29 14-34-53

**Comment by since9629** [comment]
@the22mastermind  Great job on your task, I would like to suggest a few things though:
Please commit the package-lock.json file
- Could you not disable linting rules inside components
- I have seen that there are conflicting component names. For instance, there is a component "components/Card" that has the same name as the material card component. I would suggest giving more different descriptive names to these components.

Best practices (optional)
- It would be great if you created a new folder for every new component and put in all component relevant files in that folder.
- Could you move all assets like CSS, sass, images, etc... into the /assets directory and then group them by type(e.g img, styles, icons, etc...).
- Could you also move redux related files in to a redux folder
- Could you use JSS to hold global styles(classes)- Could you use global constants to hold css colors/classes used inside presentational components, or just create a custom theme and define primary, secondary, accent, warn, etc... colors; then, use theme.palette.[YOUR COLOR] to style your components with any of those predefined colors. You can access the theme globally on any presentational component by using useTheme() hook.
- Could you use theme.spacing() to define padding and margins. it is responsive across different screens.
- I have noticed that you are using icons from react-icons. Since we're already using material-ui, could we also use material-ui icons?
- Could you not use native HTML elements where native react/material components can be used. For instance, use `<Link>` instead of `<a>`

**Review by since9629 — APPROVED** [review]

**Review by Muhire-Josue — APPROVED** [review]
Great job

**Review by Ugizwenayo-Divine — APPROVED** [review]
well done

**Review by descholar-ceo — APPROVED** [review]
LGTM

**Comment by kipropbrian** [comment]
Good job @the22mastermind, add a screenshot to your PR next time. It helps. Good job, it looks nice.

**Pull request merged**